includes/class-wc-product-variable.php

```diff
@@ -183,6 +135,45 @@ public function get_price_html( $price = '' ) {
 	}
 
 	/**
+	 * Return a products child ids.
+	 *
+	 * @return array Children ids
+	 */
+	public function get_children( $context = 'view' ) {
```

Comment:
Since we are deprecating `$visible_only` on 2.6's `get_children`, and not a `$context` parameter, can we rename the parameter (and deprecation check) to use `$visible_only` instead?